crates/criticaltrust/src/keys/pair.rs

```diff
@@ -16,5 +16,8 @@ pub trait KeyPair {
     fn public(&self) -> &PublicKey;
 
     /// Sign the provided data with this key pair, returning the bytes of the signature.
-    fn sign(&self, data: &PayloadBytes<'_>) -> Result<SignatureBytes<'static>, Error>;
+    fn sign(
```

Comment:
This signature was determined via following a clippy lint. I originally had `async fn sign(...)` but it recommended this.

Reply:
Clippy is such a wonderful co-pilot.